pkg/fleet/internal/exec/installer_exec.go

```diff
@@ -69,8 +68,15 @@ func (i *InstallerExec) newInstallerCmd(ctx context.Context, command string, arg
 }
 
 // Install installs a package.
-func (i *InstallerExec) Install(ctx context.Context, url string, _ []string) (err error) {
-	cmd := i.newInstallerCmd(ctx, "install", url)
+func (i *InstallerExec) Install(ctx context.Context, url string, args []string, forceInstall bool) (err error) {
+	var cmdLineArgs = []string{url}
+	if forceInstall {
+		cmdLineArgs = append(cmdLineArgs, "--force")
+	}
+	if len(args) > 0 {
+		cmdLineArgs = append(cmdLineArgs, fmt.Sprintf("--install_args \"%s\"", strings.Join(args, ",")))
```

Comment:
should these be separate arg list elements?

```suggestion
		cmdLineArgs = append(cmdLineArgs, "--install_args")
		cmdLineArgs = append(cmdLineArgs, fmt.Sprintf(`"%s"`, strings.Join(args, ",")))
```